### PR TITLE
Extend Kusama coverage for learn-staking doc

### PR DESCRIPTION
**Pull request by alfarok**
Part of the #4110 initiative.

### CODE REVIEW

File: docs/learn/learn-staking.md [hunk 1]
```diff
@@ -518,22 +530,21 @@ return will be less, encouraging some users to withdraw.
 
 ### Cons of Staking
 
-- Tokens will be locked for about 28 days on Polkadot after unbonding, and seven days on Kusama.
+- Tokens will be locked for about {{ polkadot: 28 :polkadot }}{{ kusama: 7 :kusama }} days on
```
Comment (alfarok) [inline docs/learn/learn-staking.md:542]: same issue as previously discussed, requires math from multiple chain values

File: docs/learn/learn-staking.md [hunk 2]
```diff
@@ -16,9 +16,12 @@ Start your staking journey or explore more information about staking on
 [Staking Dashboard](https://staking.polkadot.network/#/overview) that makes staking much easier and
 check this
 [extensive article list](https://support.polkadot.network/support/solutions/articles/65000182104) to
-help you get started.
-{{ polkadot: You can now [stake natively with just 1 DOT and earn staking rewards](https://polkadot.network/blog/nomination-pools-are-live-stake-natively-with-just-1-dot/). :polkadot }}
-{{ kusama: All the examples presented on Polkadot apply to Kusama as well. :kusama }}
+help you get started. You can now on stake
```
Comment (profradha) [inline docs/learn/learn-staking.md]:
```suggestion
help you get started. You can now stake on 
```

File: docs/learn/learn-staking.md [hunk 3]
```diff
@@ -95,11 +98,13 @@ in-depth review of NPoS see
 :::caution Minimum Nomination to Receive Staking Rewards
 
 Although the minimum nomination intent is
-{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }},
+{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/>, :polkadot }}
```
Comment (profradha) [inline docs/learn/learn-staking.md]:
```suggestion
{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={1000000000000} filter="humanReadable"/>, :polkadot }}
```

File: docs/learn/learn-staking.md [hunk 4]
```diff
@@ -95,11 +98,13 @@ in-depth review of NPoS see
...
+{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/>, :kusama }}
 it does not guarantee staking rewards. The nominated amount has to be greater than
 [minimum active nomination](learn-nominator.md#minimum-active-nomination-to-receive-staking-rewards),
 which is a dynamic value that can be much higher than
-{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }}.
+{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/>. :polkadot }}
```
Comment (profradha) [inline docs/learn/learn-staking.md]: Changed recently to 100 DOT

```suggestion
{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={1000000000000} filter="humanReadable"/>. :polkadot }}
```

**Review by DrW3RK — APPROVED** [review]

**Comment by DrW3RK** [comment]
Can merge after the minor fixes

**Comment by DrW3RK** [comment]
There are a couple of instances of just the mention of DOT in Slashing section

File: docs/learn/learn-staking.md (hunk 3 above)
Comment (alfarok) [reply docs/learn/learn-staking.md]: Great catch!  I think I was mistakenly using Kusama's default

**Pull request merged**